app/src/layouts/pageLayout/pageBreadcrumbs/pageBreadcrumbs.scss

```diff
@@ -23,3 +23,27 @@
     margin: 0 3px;
   }
 }
+
+:global {
+  div[class^='breadcrumb__breadcrumb']:nth-child(n + 3),
```

Comment:
Please don't use `:global` and hacks to overpass css modules.
Styles should belong to the corresponding component.
The only case for using `:global` is overriding styles of a third-party library.